packages/ssz/test/unit/byType/listComposite/tree.test.ts

```diff
@@ -194,3 +194,34 @@ describe("ListCompositeType.sliceTo", () => {
     }
   });
 });
+
+describe("ListCompositeType.sliceFrom", () => {
+  it("Slice List at multiple length", () => {
```

Comment:
Would be better to split the tests into bad and good assertions. e.g. separate test for negative index.